content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,127 @@
+can also enhance their own skills and knowledge. When tasked with explaining and teaching complex concepts to others, mentors are compelled to
+develop a comprehensive understanding of the subject matter, thereby expanding their own expertise. Furthermore, mentorship provides a valuable
+opportunity to gain insights into personal communication styles, effective goal-setting strategies, and other valuable areas of personal and
+professional growth.
+
+Examples, of how to influence others:
+* Participate in or start a mentoring program at your company.
+* Actively demonstrate company values and culture through your actions.
+* Review code, documents, proposals, etc.
+* Give feedback!
+
+## Overcommunicate and bring clarity
+![](./communication.png)
+
+Miscommunication is often a source of most problems at work (and in life). Misunderstanding can result in poor performance, subpar
+quality, stressed people, or wasted time. The role of staff engineers is to bring 
+[clarity and alignment](https://alexewerlof.substack.com/p/6-ways-staffs-reach-clarity).
+
+There are several things staff engineers should do to improve communication in their teams and company:
+* Listen carefully to others and understand their problems and point of view.
+* Identify and challenge the [X/Y problem](https://xyproblem.info/).
+* Make sure the team reaches a collective understanding.
+* [Start with why](https://simonsinek.com/books/start-with-why/) and explain why we work on certain things.
+* Lead by example and improve their communication skills. Show more junior engineers this is often a key to success.
+
+## Bring calmness
+![](./calm.jpg)
+
+There are going to be some stressful situations at work. It can be either a tight deadline or an application incident. Staff engineers should
+show their experience and stay calm during the crisis. They should take a lead, communicate clearly and help with prioritization. Ultimately, their role is to facilitate the success of their team members and optimize their collective performance.
+
+Proactively preparing for potential failure scenarios is another effective strategy for bringing calmness to the organization. Staff engineers 
+can begin by conducting a thorough assessment of possible risks even before the project's commencement. During the ideation and implementation 
+phases, they should systematically review potential risks, evaluate the efficacy of existing mitigation plans, and make necessary adjustments. 
+By anticipating and preventing crises before they occur, staff engineers can minimize their impact and effectively steer their team toward success.
+
+So what to do to bring calmness to the team?
+* Perform a comprehensive risk assessment, conduct pre-mortems, and anticipate potential failure scenarios before they arise.
+* In critical situations take a lead and help the team solve the problem.
```

Comment:
https://www.ldoceonline.com/dictionary/take-the-lead-in-doing-something
`to take the lead`